### DISCUSSION TITLE
Cache volume is not writable in rootless build container

### DISCUSSION THREAD

**Discussion by MShekow** [containers/buildah#4466] [Q&A] [answered]
I'm on RHEL 8.6 and I'm running a rootless build via `podman build -f Dockerfile .` with a very simple `Dockerfile`:

```
FROM busybox:latest
RUN --mount=type=cache,id=some-cache,target=/cache touch /cache/foo
```
The `RUN` statement fails with `touch: /cache/foo: Permission denied`

Is this expected behavior? What do I need to do to get it working?

**Comment by flouthoc** [comment, marked as answer]
Hi @MShekow , Is SElinux enabled ? if yes consider adding a `RUN --mount=type=cache,id=some-cache,target=/cache,z touch /cache/foo`

**Reply by MShekow** [reply]
That worked, thank you!
